packages/appstore/src/components/CFDs/cfd-accounts.scss

```diff
@@ -0,0 +1,2450 @@
+.cfd-accounts-container {
+    &__compare-accounts {
+        cursor: pointer;
+    }
+    .cfd-accounts {
+        &__title {
+            display: flex;
+            justify-content: flex-start;
+            align-items: center;
+            & > span {
+                margin-right: 1.2rem;
+            }
+        }
+
+        &__description {
+            margin-bottom: 2.4rem;
+            a {
+                text-decoration-line: underline;
+            }
+        }
+    }
+
+    .cfd-demo-account,
+    .cfd-real-account {
+        &__accounts {
+            display: flex;
+            justify-content: start;
+            margin-top: 4rem;
+            & > div {
+                margin-right: 2.4rem;
```

Comment:
Is this to space the `div` children within this container? If yes you can use `gap: 2.4rem` in the parent class